cypress/e2e/settings/apps.cy.ts

```diff
@@ -34,6 +34,10 @@ describe('Settings: App management', { testIsolation: true }, () => {
 
 		// I am logged in as the admin
 		cy.login(admin)
+
+		// Intercept the apps list request
+		cy.intercept('GET', '*/settings/apps/list').as('fetchAppsList')
+
 		// I open the Apps management
 		cy.visit('/settings/apps')
```

Comment:
Patch was not complete because the URL changed to `/settings/apps/installed`.
```suggestion
		cy.visit('/settings/apps')

		// Wait for the apps list to load
		cy.wait('@fetchAppsList')
```